skorch/net.py

```diff
@@ -2138,6 +2135,23 @@ def __getstate__(self):
         return state
 
     def __setstate__(self, state):
+        # in skorch 0.11 -> 0.12, we made a change to parameter validation. We
+        # don't store key/vals in self._kwargs anymore, as the values were
+        # redundant and were not considered as possibly CUDA dependent. Instead,
+        # we now use the attribute '_params_to_validate', which only stores
+        # keys. The code below is to make the net backwards compatible.
```

Comment:
Please add a expiration date for this (e.g. in the next year from now or something)

Reply:
Done